gen/templates/provider.go

```diff
@@ -58,6 +60,7 @@ type FmcProviderModel struct {
 // FmcProviderData describes the data maintained by the provider.
 type FmcProviderData struct {
 	Client      *fmc.Client
+	Version		*version.Version
```

Comment:
I wonder if this is something we should implement at the client level, as it might also be useful for someone just using the client.

Reply:
Did implementation at go-fmc level. Can you please review it?

Reply:
I think this can be removed now.

Reply:
Removed.